styles/advanced-open-file.less

```diff
@@ -5,6 +5,7 @@
     .list-group {
         overflow: auto;
         flex: 2;
+        position: relative;
```

Comment:
The offsets/positions are relative to the first positioned parent, so we need this to make sure they're relative to the scrollable list.